SECURITY.md

```diff
@@ -0,0 +1,55 @@
+# Coordinated Vulnerability Disclosure Policy
+
+The Cosmos ecosystem believes that strong security is a blend of highly technical security researchers
+who care about security and the forward progression of the ecosystem and the attentiveness and
+openness of Cosmos core contributors to help continually secure our operations.
+
+> **IMPORTANT**: *DO NOT* open public issues on this repository for security vulnerabilities.
+
+## Scope
+
+| Scope                 |
+|-----------------------|
+| last release (tagged) |
+| main branch           |
+
+The latest **release tag** of this repository is supported for security updates as well as the **main** branch.
+Security vulnerabilities should be reported if the vulnerability can be reproduced on either one of those.
+
+## Reporting a Vulnerability
+
+| Reporting methods                                                 |
+|-------------------------------------------------------------------|
+| [GitHub Private Vulnerability Reporting](security/advisories/new) |
```

Comment:
Pressing on this opens this URL
<https://github.com/cometbft/cometbft-db/blob/00053c5426b140655ca3b13ee295269171983b0b/security/advisories/new>
which is a 404